translations/mod-fqm-manager/en.json

```diff
@@ -491,7 +491,7 @@
   "entityType.simple_item_details.permanent_location_id": "Permanent location UUID",
   "entityType.simple_item_details.purchase_order_line_identifier": "Purchase order line identifier",
   "entityType.simple_item_details.statistical_code_ids": "Statistical code UUIDs",
-  "entityType.simple_item_details.statistical_code_names": "Statistical code names",
```

Comment:
This violates our translation guidelines, but our POs talked it over and ran it by the BELA working group, and they landed on making it singular to keep the name consistent, since it's always singular in quite a few places, and only plural in the DB field name that users never actually see